db/seeds.rb

```diff
@@ -0,0 +1,31 @@
+require 'faker'
+
+User.create(username: "admin")
```

Comment:
always use `create!` in your seed files (in case it doesn't actually get created, then you try to launch your app and have no data. just another thing to have to debug that we can easily avoid with `create!`)